File: rentabikewtr_desktop/lib/model/servisiranja.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';
import 'package:rentabikewtr_desktop/model/rezervniDijeloviPregled.dart';

part 'servisiranja.g.dart';

@JsonSerializable()
class Servisiranja {
  String? opisKvara;
  DateTime? datumServisiranja;
  String? preduzetaAkcija;
  String? komentarServisera;
  //String? nazivBicikla;
  int? biciklID;
  List<RezervniDijeloviPregled>? rezervniDijelovi;

  Servisiranja(
      {this.opisKvara,
      this.datumServisiranja,
      this.preduzetaAkcija,
      this.komentarServisera,
      //this.nazivBicikla,
      this.biciklID,
      this.rezervniDijelovi}) {}

  factory Servisiranja.fromJson(Map<String, dynamic> json) =>
      _$ServisiranjaFromJson(json);

  /// Connect the generated [_$ServisiranjaToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ServisiranjaToJson(this);
}

File: RentABikeWTR_v1_7.Model/Servisiranja.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RentABikeWTR_v1_7.Model
{
    public class Servisiranja
    {
        public int ServisiranjeId { get; set; }
        public string OpisKvara { get; set; }
        public DateTime DatumServisiranja { get; set; }

        public string? PreduzetaAkcija { get; set; }
        public string? KomentarServisera { get; set; }
        public int? BiciklID { get; set; }
        public List<RezervniDijelovi>? RezervniDijelovi { get; set; }
    }
}
